nexus-sdk-js/src/views/ElasticSearchView/index.ts

```diff
@@ -177,6 +192,33 @@ class ElasticSeachView {
       throw new ViewQueryError(error.message);
     }
   }
+
+  /**
+   * Query aggregations in elastic search
+   * using an elastic search Aggregation Query (as a JS Object) as input
+   * https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html
+   *
+   * @param {Object} elasticSearchAggregationQuery
+   * @returns {Promise<ElasticSearchViewAggrigationResponse>}
+   * @memberof ElasticSeachView
```

Comment:
Oh so many renames after fixing the original typo

Reply:
F2 is your friend